automobile/buildings.md

```diff
@@ -63,11 +64,12 @@ Building {
 - `name`: Could contain the building name.
 - `floorHeight`: Defines the height of one floor.
 - `floorNumber`: Defines the number of floors (excluding roof).
+- `startingFloor`: Defines the number of floors between the ground and the first floor.
```

Comment:
I would say "Defines the floor number for the "ground floor" of the building, as not all buildings start at the ground floor.".